pkg/features/kube_features.go

```diff
@@ -133,6 +133,12 @@ const (
 	// Taint nodes based on their condition status for 'NetworkUnavailable',
 	// 'MemoryPressure', 'OutOfDisk' and 'DiskPressure'.
 	TaintNodesByCondition utilfeature.Feature = "TaintNodesByCondition"
+
+	// owner: @jsafrane
+	// alpha: v1.8
+	//
+	// Enable mount propagation of HostPath volumes.
```

Comment:
Is it really tied to just HostPath volumes?

Reply:
fixed